public/_redirects

```diff
@@ -216,5 +216,12 @@ https://docs.cert-manager.io/* https://cert-manager.io/docs/:splat 302!
 /docs/release-notes/* /docs/releases/release-notes/:splat 301!
 /docs/installation/supported-releases/ /docs/releases/ 301!
 
+## The /docs/installation/upgrading/ and /docs/release-notes/ pages do not exist anymore
```

Comment:
The new page for upgrading is https://cert-manager.io/docs/installation/upgrade/, right?

That problem was introduced here https://github.com/cert-manager/website/commit/d34d97811cf3251d6da35d12f2fd88ba831a70bb#diff-f70b82b8d6c19e4cd592d939ae778b3ec6a3cf911941f5baab8b73c6dd7c1e25

So we should either revert that change or add a redirect from `/docs/installation/upgrading/` > `/docs/installation/upgrade/.

Reply:
It really depends on whether you want info about the new versions that have been released or about how to upgrade to a new version.